docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -125,9 +127,9 @@ Ensure you stop Besu before running the command.
 
 ###  Subcommand not working
```

Comment:
drop H3s to H2s to follow H1

Reply:
These are under the troubleshoot section (H2), so should be under (H3)

Reply:
These are under the troubleshoot section, so should be